### PR TITLE
cb_channels: get system-wide channels list

### PR DESCRIPTION
**Pull request by onnet**

### CODE REVIEW

File: applications/crossbar/src/modules/cb_channels.erl [hunk 1]
```diff
@@ -508,3 +511,10 @@ maybe_intercept(Context, CallId, TargetType, TargetId) ->
     lager:debug("attempting to move ~s to ~s(~s)", [CallId, TargetId, TargetType]),
     kz_amqp_worker:cast(API, fun kapi_metaflow:publish_req/1),
     crossbar_util:response_202(<<"intercept initiated">>, Context).
+
+-spec get_account_id(cb_context:context()) -> ne_binary().
+get_account_id(Context) ->
+    case cb_context:account_id(Context) of
+        'undefined' -> <<"all">>;
+        AccountId -> AccountId
+    end.
```
Comment (lazedo) [inline applications/crossbar/src/modules/cb_channels.erl]: @onnet please add a system config to enable this search, should default to `false`. when you potentially have thousands of channels you don't want admins to inadvertently use this option

**Comment by onnet** [comment]
Sorry, I'm out of ideas what to feed to dialyzer anymore...

**Comment by lazedo** [comment]
@onnet i believe this is not because of this pr. investigating

**Comment by lazedo** [comment]
@onnet this one is weird...,
so there was a commit yesterday that changed the channel record but didn't touch ecallmgr_fs_channels so dialyze wasn't run for this file. your pull request changes ecallmgr_fs_channels so dialyze checks it.

this dialyzer warning is a false positive and we needed to change the dialyzer script.
please rebase to get the script changes.

**Pull request merged**